.github/workflows/integration.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         cluster: ["opensearch"]
         secured: ["true"]
         entry:
-          - { opensearch_version: 2.6.0 }
+          - { opensearch_version: 2.5.0 }
```

Comment:
Why did you change this back?

Reply:
The tests are passing in my local environment, but not the workflow. So I wanted to see if the problem was with the version. Turns out it's not, I will change it back.

I am suspecting the problem is with the circuit breaker error. I am investigating.